libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -642,7 +642,7 @@ void NavEKF2_core::readGpsData()
         } else {
             // report GPS fix status
             gpsCheckStatus.bad_fix = true;
-            hal.util->snprintf(prearm_fail_string, sizeof(prearm_fail_string), "Waiting for 3D fix");
+            hal.util->snprintf(prearm_fail_string, sizeof(prearm_fail_string), "Waiting for GPS fix %u", gps.status_arm_min());
```

Comment:
Since we're not requiring enhanced fix, this can be reverted right?

Reply:
Yea you are right. Done.